app/models/form_profiles/va_526ez.rb

```diff
@@ -167,9 +167,7 @@ def prefill_base_class_methods
 
   def initialize_form526_prefill
     VA526ez::Form526Prefill.new(
-      # any form that has a startedFormVersion (whether it is '2019' or '2022') will go through the Toxic Exposure flow
-      # '2022' means the Toxic Exposure 1.0 flag.
-      started_form_version: Flipper.enabled?(:disability_526_toxic_exposure, user) ? '2022' : nil,
+      started_form_version: Flipper.enabled?(:disability_526_toxic_exposure, user) ? '2022' : '2019',
```

Comment:
we can probably remove the check here and just put "2022" since prefill is only called on new forms

Reply:
y'all are making my future merge conflicts too easy! 😂
https://github.com/department-of-veterans-affairs/vets-api/pull/20043/files